agent/handlers/task_server_setup_integ_test.go

```diff
@@ -0,0 +1,123 @@
+//go:build integration
+// +build integration
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// This function starts the server and listens on a specified port
+func startServer() *http.Server {
+	router := mux.NewRouter()
+	registerFaultHandlers(router, nil, nil)
+	server := &http.Server{
+		Addr:    ":5932",
+		Handler: router,
+	}
+
+	go func() {
+		if err := server.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+			fmt.Printf("ListenAndServe(): %s\n", err)
+		}
+	}()
+	return server
+}
+
+// This function shuts down the server after the test
+func stopServer(server *http.Server) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	if err := server.Shutdown(ctx); err != nil {
+		fmt.Printf("Server Shutdown Failed:%+v", err)
```

Comment:
It would be better to `t.Logf`

Reply:
Addressed in [1e79b26](https://github.com/aws/amazon-ecs-agent/commit/1e79b265acb772ff7092b24ba6fc1551d1fda3cf)